exercises/pangram/src/main/java/Pangrams.java

```diff
@@ -0,0 +1,6 @@
+public class Pangrams {
```

Comment:
Solid.  All those points totally click for me.  I'm sold on the Function Object idea, myself.

I like the idea of renaming the class, too.

Thanks for taking the time to talk this out, Frida.